pom.xml

```diff
@@ -2043,6 +2043,10 @@ flexible messaging model and an intuitive client API.</description>
         <module>pulsar-client-auth-sasl</module>
         <module>pulsar-config-validation</module>
 
+        <!-- tiered-storage related modules -->
+        <module>tiered-storage</module>
+        <module>jclouds-shaded</module>
+
```

Comment:
Do we need to add this?

Reply:
When githup was packaged, these two modules were not packaged